src/UI/Component/Chart/Speedo/Speedo.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/* Copyright (c) 2017 Ralph Dittrich <dittrich@qualitus.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Chart\Speedo;
+use ILIAS\UI\Component\Component;
+
+/**
+ * Interface Speedo
+ * @package ILIAS\UI\Component\Chart\Speedo
+ */
+interface Speedo extends Component {
+
+    /**
+     * Get goal value
+     *
+     * This value is used as 100%.
+     * This value will always returned "raw" because it is used to calculate the
+     * percentage values of score, minimum and diagnostic.
+     *
+     * @return int|float
+     */
+    public function getGoal();
+
+    /**
+     * Get minimum goal value

```

Comment:
minimum or minimum goal. Please decide if you call those values either: Minimum Goal and Maximum Goal or just minimum and maximum (on the interface and in the descriptions).